website/content/en/preview/faq.md

```diff
@@ -222,6 +222,14 @@ error: error validating "provisioner.yaml": error validating data: ValidationErr
 
 The `startupTaints` parameter was added in v0.10.0.  Helm upgrades do not upgrade the CRD describing the provisioner, so it must be done manually. For specific details, see the [Upgrade Guide]({{< ref "./upgrade-guide/#upgrading-to-v0100" >}})
 
+## Upgrading Kubernetes Cluster
+
+### How do I upgrade an EKS Cluster with Karpenter?
+
+When upgrading an Amazon EKS cluster, [Karpenter's Drift feature]({{<ref "./concepts/deprovisioning#drift" >}}) can automatically upgrade the Karpenter-provisioned nodes to stay in-sync with the EKS control plane. Karpenter Drift currently needs to be enabled using a [feature gate]({{<ref "./concepts/settings#featuregates" >}}). Karpenter's default [AWSNodeTemplate `amiFamily` configuration]({{<ref "./concepts/node-templates#specamifamily" >}}) uses the latest EKS Optimized AL2 AMI for the same major and minor version as the EKS cluster's control plane. Karpenter's AWSNodeTemplate can be configured to not use the EKS optimized AL2 AMI in favor of a custom AMI by configuring the [`amiSelector`]({{<ref "./concepts/node-templates#specamiselector" >}}). If using a custom AMI, then the trigger to rollout new nodes is the publication of a new AMI matching the amiSelector, or a change to the amiSelector field.
+
+Start by [upgrading the EKS Cluster control plane](https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html). After the EKS Cluster upgrade completes, Karpenter's Drift feature will detect that the Karpenter-provisioned nodes are using EKS Optimized AMIs for the previous cluster version, and [automatically cordon, drain, and replace those nodes]({{<ref "./concepts/deprovisioning#controlflow" >}}). To support pods moving to new nodes, follow Kubernetes best practices by setting appropriate pod [Resource Quotas](https://kubernetes.io/docs/concepts/policy/resource-quotas/), and using [Pod Disruption Budgets](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/) (PDB). Karpenter's Drift feature will spin up replacement nodes based on the pod resource requests, and will respect the PDBs when deprovisioning nodes.
```

Comment:
Maybe consider moving the line that says, "To support pods moving to new nodes, follow..." into a `{{% alert title="Note" color="primary" %}}` section

Reply:
I suggest a separate pass to consider how to highlight the message, as it applies generally to Deprovisioning. For now, I suggest don't highlight here, as it may imply that it's only relevant to cluster upgrades.

Reply:
Sure, I didn't realize that it was meant to apply to all deprovisioning. The fact that it's in the same block as a statement about cluster upgrade (in my mind) automatically implies that it is related to that same topic and doesn't apply generally. Still fine though, we can take another pass at it later.

Reply:
You're right, the text is currently in the context of cluster upgrade. I mean we should highlight the point in a more generic location that applies to all deprovisioning. That I'd rather do in a separate pass.